src/services/ckan/datasetList.ts

```diff
@@ -6,16 +6,47 @@ import axios from 'axios';
 import { PackageSearchOptions, PackageSearchResult } from './types/packageSearch.types';
 import { constructCkanActionUrl, mapCKANPackageToDataset } from './utils';
 
+const mapFacetToLabel = (facet: string) => {
```

Comment:
Maybe we could have this as mapping instead of switch case, should perform better